src/store/plugins/ledgerConnection.js

```diff
@@ -1,8 +1,10 @@
 import TransportU2F from '@ledgerhq/hw-transport-u2f';
 import Ae from '@aeternity/ledger-app-api';
 import { Crypto } from '@aeternity/aepp-sdk/es';
+import { isSupported } from 'u2f-api';
 
 export default async (store) => {
+  isSupported().then(supported => store.commit('setLedgerSupported', supported));
```

Comment:
please, use `await` here, also you can take `isSupported` method from `TransportU2F` class